src/test/resources/features/foi/ConsiderAndDraft.feature

```diff
@@ -0,0 +1,55 @@
+@FOIConsiderAndDraft @FOI @FOIWorkflow
+Feature: Consider and Draft
+
+  Background:
+    Given I am logged into "CS" as user "FOI_USER"
+    When I create a "FOI" case and move it to the "Consider and Draft" stage
+    And I load and claim the current case
+
+  Scenario: User is able to complete the Consider and Draft stage
+    And I select that the case "Is" in the correct drafting team
+    And I click the "Continue" button
+    And I select that the case "Doesn't" require a contribution
+    And I click the "Continue" button
+    And I select "Full disclosure" as the response type
+    And I click the "Continue" button
+    And I click add documents
+    And I choose the document type "Draft response"
+    And I upload a file of type "docx"
+    And I click the "Continue" button
+    Then the case should be moved to the "Approval" stage
+
```

Comment:
For this we should move the click buttons into the steps above, maybe rename them so they are move descriptive of the users goal than the specific action on the application, and the 3 document steps should all be one thats like 'I add a Draft Response document'